CODEOWNERS

```diff
@@ -0,0 +1,17 @@
+* @stuhlmueller
+/notebooks/ @brachbach
+/tests/ @stuhlmueller
+/tests/test_scales.py @djthorne
+/tests/test_metaculus.py @brachbach
+/tests/test_logistic.py @djthorne
+/scripts/ @brachbach
```

Comment:
@stuhlmueller @djthorne knows way more about `/scripts/` than I do, would recommend making him the ownder